components/Footer.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function Footer() {
+  return (
+    <div className="border-2   flex bg-[white] justify-center bottom-0">
```

Comment:
![image](https://user-images.githubusercontent.com/45912180/200631435-eb5a787d-b1c2-4fcc-8ce7-e02f59e2d256.png)

Footer should stick to the bottom if content doesn't fill the page

Reply:
Also a little padding can be added

Reply:
> Also a little padding can be added

Are we planning it to keep its position fixed ? if so the footer component will be visible on the view port taking up 1/4 th of the viewport space, my idea was to keep the footer at the end of the content (following DOM flow) so that the footer is only shown when the user hits the end of the content , and content has to be present on the first page load , so if im not wrong this wouldnt be a problem , let me know what you think .

Reply:
That works, so what we'll do it make sure our content is atleast of 100vh (I've done it with 'bidding window') and we'll see how it turns out.

Reply:
> That works, so what we'll do it make sure our content is atleast of 100vh (I've done it with 'bidding window') and we'll see how it turns out.

Great 👍